src/components/Navigator/modes/messages.ts

```diff
@@ -0,0 +1,57 @@
+import { defineMessages } from "react-intl";
+
+const messages = defineMessages({
+  addCategory: {
+    defaultMessage: "Add Category",
```

Comment:
Buttons in the UI use "Create <object>" convention and I think these actions should use the same convention. BTW once you're inside the "Create" view the page header usually says "Add <object>" - is this on purpose?

Reply:
OK it's a bug in the Customers section - it should say "Create Customer".